src/main/java/org/sample/StringsMixed.java

```diff
@@ -0,0 +1,118 @@
+package org.sample;
+
+import java.lang.reflect.Method;
+import java.util.concurrent.TimeUnit;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+@State(Scope.Thread)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@Warmup(iterations = 30, time = 2, timeUnit = TimeUnit.SECONDS)
+@Measurement(iterations = 10, time = 2, timeUnit = TimeUnit.SECONDS)
+@Fork(3)
+public class StringsMixed {
+    private static final String PREFIX = "a";
+    private static final String SEPARATOR = ";";
+    private static final String FORMAT =
+            PREFIX + "%d" + SEPARATOR + "%s" + SEPARATOR + "%b";
+    private long time;
+    private double nanoTime;
+    private int LENGTH;
+    private boolean oddTime;
+
+    @Setup
+    public void setup() {
+        time = System.currentTimeMillis();
+        nanoTime = System.nanoTime();
+        oddTime = time % 2 == 1;
+        String expected = plain();
+        LENGTH = expected.length();
+        for (Method method : StringsMixed.class.getDeclaredMethods()) {
```

Comment:
Is this a kind of Junit test approach?

Reply:
It is a sanity check to test that we are generating the same Strings.

Reply:
I see. Interesting. Saves us a framework. Thanks for explaining.